src/c++/perf_analyzer/command_line_parser.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright 2022-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Update `--service-kind` help messages to include `openai`

Reply:
good catch! Also added `endpoint` in a few places in help and cleaned up the help a little